src/Components/ActivityCards/ActivityCards.scss

```diff
@@ -0,0 +1,56 @@
+@import  '../../styles/variables.scss';
+
+.activities-info{
+    padding: 2%;
+    background-color:$light;
+
+    .activity-container{
+        height: 50vh;
+        background-color:$light;
+        display: grid;
+        grid-template-columns: repeat(5, 1fr);
+        grid-gap: 25px;
+        justify-content: center;
+
+        .inner-activity-container{
```

Comment:
What is the purpose of nesting here?

Reply:
I know it has to with SCSS and child /parent elements but I also and wondering about the advantages of doing it if you could elaborate @Atos20

Reply:
of course, sorry for not explaining this before but SCSS syntax requires you to organize everything using the parent-child relation. This concept actually made me understand react components a little bit faster because it is the same idea behind nesting components using JSX syntax. 

The advantage I have found of using this syntax is that it makes locating elements faster and easier when styling components. It truly makes you think of the child-parent relationship between HTML elements.

Reply:
Thanks for this clarification!